src/client/components/organism/navbar/styles.scss

```diff
@@ -65,7 +65,10 @@
       }
     }
   }
-
+  &__burgerButton {
+   margin:auto;
+   margin-right: 0;
+  }
```

Comment:
let's leave a empty line between CSS selectors, for better comprehension in future